docs/tmpl/navbar.tmpl

```diff
@@ -14,7 +13,7 @@
                         <span class="icon-bar"></span>
                         <span class="icon-bar"></span>
                     </button>
-                    <a class="navbar-brand hidden-sm" href="./index.html">vis.js</a>
+                    <a class="navbar-brand hidden-sm" href="./index.html">vis.js!</a>
```

Comment:
!?

Reply:
😆 I used that to check that the partial templates were indeed loading. Overisight, hereby fixed.